codegenerator/cli/templates/static/codegen/src/ContractInterfaceManager.res

```diff
@@ -5,8 +5,8 @@ interfaces and their related contracts and addresses.
 Used to work with event topic hashes and in the parsing of events.
 */
 type interfaceAndAbi = {
-  interface: Ethers.Interface.t,
   abi: Ethers.abi,
+  sighashes: array<string>,
 }
```

Comment:
This module doesn't look very useful anymore, probably we should refactor it in the future.

Reply:
Agreed!